src/shims/manual/mscorlib.forwards.cs

```diff
@@ -14,9 +14,6 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.NonRandomizedStringEqualityComparer))]
```

Comment:
This one should not be needed either. I am not touching it because of things are pretty confused around it. Related to #24938